sdk/tables/azure-data-tables/src/main/java/com/azure/data/tables/implementation/models/TableEntityQueryResponse.java

```diff
@@ -17,7 +17,7 @@
  * The properties for the table entity query response.
  */
 @Fluent
-public final class TableEntityQueryResponse implements JsonSerializable<TableEntityQueryResponse> {
+public final class TableEntityQueryResponse extends TablesJacksonSerializer implements JsonSerializable<TableEntityQueryResponse> {
```

Comment:
From an API perspective, I think it would be better for TableEntityQueryResponse` to have a constant property of a `TablesJacksonSerializer` rather than extending it. Every time an instance of `TableEntityQueryResponse` is instantiated it will create a new instance of `JacksonAdapter` as super types are initialized on creation. This will have a large overhead as this will create new `ObjectMapper`'s each time, which is expensive.